src/calculateRentalCost.js

```diff
@@ -28,7 +28,28 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalRent = 0;
+  const minimalDiscount = 20;
+  const basicDiscount = 50;
+  const dailyRent = 40;
+
+  if (days < 3) {
+    totalRent += dailyRent * days;
+
+    return totalRent;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalRent += dailyRent * days - minimalDiscount;
```

Comment:
same

Reply:
no, he can't! look closer at his code and especially on the var `totalRent` he used it to collect the results)